addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -274,15 +345,33 @@ img.o_forum_avatar_big {
 .website_forum {
     margin-bottom: $spacer;
 }
+.popover {
+    box-shadow: 0 2px 5px $gray-400;
+
+    .popover-header {
+        background-color: tint-color($primary, 80%);
+        border-bottom: 0;
+        color: $primary;
+    }
+}
 
 .popover.o_wforum_bio_popover_container {
     max-width: 552px;
 }
 
-.o_wforum_forum_card_bg {
-    background-image: linear-gradient(99deg, map-get($theme-colors, 'secondary') 10%, darken(map-get($theme-colors, 'secondary'), 10%) 90%);
+#o_wforum_forums_index_list .card {
+    text-decoration: none;
```

Comment:
you can use multiple utility classes instead: `text-decoration-none`,  `transition-base`...